tracer/test/Datadog.Trace.TestHelpers/TestHelper.cs

```diff
@@ -661,6 +675,86 @@ protected void EnableTelemetry(bool enabled = true, int? standaloneAgentPort = n
             Assert.Equal(expectedServiceVersion, span.Tags.GetValueOrDefault(Tags.Version));
         }
 
+        protected void SetupRcmConfiguration(string path = null)
+        {
+            if (path == null)
+            {
+                path = Path.GetTempFileName();
+                File.Delete(path);
```

Comment:
I think `Path.GetTempFileName` creates an empty file, ready to write content into it 🤔  
So, you can use it right away, instead of deleting/creating dir/and deleting file again?

Reply:
I want to take the file name as directory, in order to place the remote config file inside (delete, create directory)

Reply:
I understand the implementation, but I'm not sure I understand the reason 🤔 
It removed anyway during `Dispose` process, right?
And it's inside of the `Temp` folder, that a big mess, anyway?

Reply:
It is a way of getting a random dir name for every test